docs/changelog.md

```diff
@@ -37,6 +37,68 @@ It doesn't include
+  embedded browser or the external browser. (PR #4679)
+* Added a server-compatibility notice for self-hosted orgs running Zulip
+  Server versions before 2.0.0. (PR #4750)
+* Stream settings screen now has a "subscribe" button. And, when
+  unsubscribed, useless toggle switches ("Pinned", "Muted", and
+  "Notifications") are hidden. (#4715)
+* Updated the primary app color to match the new logo. (PR #4544)
+* Fixed a bug on Android where the app sometimes opened to an unexpected
+  narrow when opening the app from the background. (#4758)
+* Fixed a glitch where the org URL input screen would flash by before the
+  authentication screen, on some app startups or when the user tapped a
+  logged-out account in the account picker. (#4604)
+* Fixed misconfigurations causing the Play Store to not offer the Android
+  app on devices that don't have cameras (#4722) or WiFi capability (PR
+  #4761).
+* Continued work toward enabling Android share-to-Zulip. (#4514)
+* Unclear, little-used "create group PM" option in the compose box removed.
+  (PR #4777)
+* Miscellaneous UI improvements. (#4734; PRs #4717, #4749, #4716, #4697,
+  #4686, #3517, #3395)
+
+Plus, like every release, numerous other fixes and improvements for your
+Zulip experience.
+
+
+### Highlights for developers
+
+* Extended Jest coverage to include Android-only codepaths. (#4700)
+* All `yarn` dependencies upgraded as far as possible under semver
+  constraints, as of 2021-06-07. (PR #4789)
+* Fixed new iOS build failures. (PRs #4721, #4634)
+* `restart` events can now cause updates to `zulipVersion` and
+  `zulipFeatureLevel` in `state.accounts[0]`. (PR #4707)
+* Increased test coverage of our storage logic, and continued cleaning up
+  `redux-persist`. (#4709)
+* Miscellaneous testing improvements. (PRs #4760, #4753)
+* Fixed a bug where `CaughtUp` state was being wrongly overwritten. (PR
+  #4698)
+* In dev mode only, a user-facing string passed to a `_: GetText` will now
+  be highlighted in the UI if it doesn't have an entry in
+  `messages_en.json`. (#4728)
+* Cleaned up some code for particular API endpoints. ( PR #3955)
+* Bumped `@zulip/shared` to v0.0.5. (PR #4680)
+* Added a migration that should have been released with v27.162. (PR #4689)
+* Improved documentation for the release process. (PR #4690)
+
+* Resolved issues (latest to earliest): PR #4794, PR #4707, PR #4789, PR
+  #4520, #4766, PR #4787, PR #4785, PR #4784, PR #4780, PR #4786, PR #4721,
+  PR #4779, PR #4778, PR #4334, PR #4727, PR #4574, PR #4737, #4715, PR
+  #4777, PR #4774, PR #4709, PR #4750, #4765, PR #4730, #4758, PR #4761, PR
+  #4762, PR #4731, PR #4717, PR #4700, PR #4710, PR #4760, PR #4749, PR
+  #4736, PR #4667, PR #4755, PR #4753, #4722, PR #4514, PR #4741, #4604, PR
+  #4728, PR #4739, PR #4732, PR #3955, #3540, #4006, #4323, #4734, #4264, PR
+  #4716, PR #4711, PR #4634, PR #4699, PR #4697, PR #4703, PR #4680, PR
+  #4544, PR #4698, PR #4686, PR #4690, PR #4678, PR #4689, #3517, #3395
```

Comment:
So the way I've been compiling this "Resolved issues" list is that each item in it really is an *issue*, i.e. either a bug or a feature request; when I list a PR, it's because we didn't file a GitHub issue for it.

So for example, taking from the top of this list:
* #4794 wouldn't be on that list
* #4707 would, as it implements a feature
* #4789 wouldn't
* #4520 wouldn't
* #4766 wouldn't, even though it's a GitHub issue -- it was really a code review, reified as a GitHub issue just to help with task-tracking
* #4787 wouldn't. Though there might be an item above saying we tried out Dependabot, and mentioning #4787.
* #4785, #4784, #4780, #4786 wouldn't -- just automated upgrades
* #4721 would, as it fixes a build issue. Oh but it'd go closer to the end of the list, because it was merged May 6.
* #4779, #4778, #4334, #4727, #4574, #4737 wouldn't -- Dependabot upgrades, and the change to enable it
* #4715 definitely would -- a buggy bit of UI
* (OK, that gives the idea.)

It wouldn't be unreasonable to have a list that includes all kinds of changes, even those that aren't issues. Possibly it'd be less work to compile that way -- though I think probably not, as it makes the list quite a bit longer.

Definitely quite reasonable to experiment with different ways of writing these notes.

If we do make this list broader, though, it should have a correspondingly different heading. "Resolved issues and merged PRs"?

Reply:
Would the many muted-user PRs all get listed here? The umbrella issue, #4655, is quite near to being resolved but hasn't been yet.

Reply:
I think in that situation it'd be OK to say something like "most of #4655".

But also, that is perhaps a hint that it's time to close #4655 as resolved, and file new issues for specific known things that are missing :wink:

Reply:
> I think in that situation it'd be OK to say something like "most of #4655".

Done in my next revision 🙂